src/Tuple.ts

```diff
@@ -50,8 +49,10 @@ export const dup: <A>(x: A) => [A, A] = fork([identity, identity])
  *
  * @since 0.12.0
  */
-export const toFst = <A, B>(f: (x: A) => B): ((x: A) => [B, A]) =>
-  fork([f, identity])
+export const toFst =
+  <A, B>(f: (x: A) => B) =>
+  (x: A): [B, A] =>
+    create([f(x), x])
```

Comment:
Or should be just `[f(x), x]`? I saw you've had opposite refactor in d4593459f2b21375c0a863a6a911ce04b654259f, any particularly reason?

Reply:
They all do the same thing, sometimes there's nothing more to it than me experimenting with different, new approaches. :slightly_smiling_face: 

`create`'s purpose is in circumstances in which you need to help the type system understand you're creating a tuple without needing to manually annotate it or use `as const`, but this doesn't apply here where the return type is annotated.

Reply:
Understood, I should have mentioned that I came from a hot path optimization aiming, thinking more I realize it would be better done locally instead.

Thanks for the explanation tho.